python/graphstorm/model/utils.py

```diff
@@ -537,7 +537,13 @@ def load_pytorch_embedding(emb_path, part_policy, name):
 
 def save_pytorch_embeddings(emb_path, embeddings, rank, world_size,
     device=th.device('cpu'), node_id_mapping_file=None):
-    """ Save embeddings through pytorch a distributed way
+    """ Save node embeddings as pytorch tensors in a distributed way.
+
+        The input node `embeddings` are stored in Partition Node ID space.
+        By default, `save_pytorch_embeddings` will shuffle the order of
```

Comment:
i don't think it shuffles node embeddings by default because the default value of `node_id_mapping_file` is None.